persistent-mongoDB/Database/Persist/MongoDB.hs

```diff
@@ -131,7 +131,9 @@ import Data.Text (Text)
 import qualified Data.ByteString as BS
 import qualified Data.Text.Encoding as E
 import qualified Data.Serialize as Serialize
-import Web.PathPieces (PathPiece (..))
+import Web.PathPieces (PathPiece(..))
+import Web.HttpApiData (ToHttpApiData(..), FromHttpApiData(..))
+import Web.HttpApiData.Internal (parseUrlPieceWithPrefix, readEitherTextData, parseUrlPieceMaybe)
```

Comment:
Thanks for bringing this up! For the first version of `http-api-data` this was intentional, since I didn't know which helpers I should export and by what name (I do not particularly like current naming).
I have opened fizruk/http-api-data#18 to discuss this.